functions.php

```diff
@@ -502,3 +502,19 @@ function shiro_add_slug_body_class( $classes ) {
 	return $classes;
 }
 add_filter( 'body_class', 'shiro_add_slug_body_class' );
+
+/**
+ * Allow span and em tags into titles.
+ *
+ * @param string $title Post title.
+ * @return string
+ */
+function the_safe_title( string $title ): void {
```

Comment:
I'd consider putting this in `inc/template-functions.php` or `inc/template-tags.php` and changing the name to `wmf_safe_title()` or something similar for consistency.